tasks/main.yml

```diff
@@ -10,6 +10,22 @@
   changed_when: false
   register: helm_existing_version
 
+- name: Set Helm version to latest if none ist set
+  set_fact:
+    helm_version: "latest"
```

Comment:
I'd rather just tell users to set to `latest` (or heck, we could just default to `latest` in the defaults), instead of having this be a 'magic' variable that is set if nothing is set.

Can you set `helm_version: latest` in defaults, update the line about it in the README to say "can set a specific version or just use `latest` to get the latest version", or something like that?